features/220-cms-import-asset/api.md

```diff
@@ -0,0 +1,183 @@
+# User Requirement
+
+- Import records with skygear asset field
+- Allow cms user to upload file from cms client, when importing records
+- In the csv, the value of asset field would be the file name of uploaded files
+
+# Overview
+
+Summary of design decision here.
+
+### Keep the cms import one single API call
+
+Import feature for each project can be very different if we want to make a
+cutomized cms for the client, due to the different work flow of the client and
+the different target user.
+
+At the current state, CMS provides the import function as a whole feature,
+meaning if the client requires customization beyond what we provided, it is
+likely the developer need to implement one from scratch themselves.
+
+Thus I HIGHLY recommend to keep this feature as simple as possible.
```

Comment:
I don't get the logic behind this *Thus*, why this feature should be as simple as possible if *"it is likely the developer needs to implement one from scratch themselves"*?

If this section is intended to be the overview of Design Goal of the feature, it is quite confusing...

From what was written, my guess is, it meant to say there are two design goal:

1. Customization beyond what we provided will be developed by Skygear users from scratch themselves, so there should be some APIs to facilitate that, a.k.a. make it easy for users to develop import feature from scratch.
1. This CMS frontend is not meant to make it possible to use with every customization in the world, so the intended use case are: `[blank]`